.github/FUNDING.yml

```
@@ -1 +1,2 @@
-open_collective: polymc
+# Commented out until further notice
+# open_collective: polymc
```

We have https://opencollective.com/prismlauncher
Even if you can't donate yet, might as well add the link for when it's ready.
```suggestion
open_collective: prismlauncher
```